bld/clib/startup/a/cstrt386.asm

```diff
@@ -202,9 +202,17 @@ ENV_SEG equ     2ch
         shr     eax,16                  ; get top 16 bits of eax
         cmp     ax,'DX'                 ; if top 16 bits = "DX"
         jne     not_pharlap             ; then its pharlap
+        ;cmp    ebx,'12aJ'              ; - if ebx (version number) is 12aJ
+        cmp     ebx,4A613231h           ; - if ebx (version number) is 12aJ
```

Comment:
This appears to be the only version number ever used by the FM TOWNS version of Phar Lap, it at least works with the built-in version on the last version of the OS.